app/views/teaching_events/show.html.erb

```diff
@@ -22,27 +22,19 @@
     </div>
   </header>
 
-  <section class="event-info">
-    <% if @event.message %>
-      <div class="event-alert">
-        <%= tag.p(@event.message) %>
-      </div>
-    <% end %>
+  <div class="container teaching-event__info">
+    <article>
+      <%= render(partial: "teaching_events/show/event-summary") %>
+      <%= render(partial: "teaching_events/show/attending-training-providers") %>
+      <%= render(partial: "teaching_events/show/scribble") if @event.scribble_id.present? %>
+      <%= render(partial: "teaching_events/show/how-to-attend") if is_a_train_to_teach_event?(@event) %>
```

Comment:
Are we replicating the other 'how to attend' variants somewhere for non TTT events?

https://github.com/DFE-Digital/get-into-teaching-app/blob/master/app/views/events/show.html.erb#L84-L92

Also we only offer this currently if the event is open, has a web feed id and is TTT (the 'pending' conditional in there is so that the internal event previews work correctly -- speaking of which, we may need to make a ticket to update that to use the new event view):

https://github.com/DFE-Digital/get-into-teaching-app/blob/master/app/views/events/show.html.erb#L66
https://github.com/DFE-Digital/get-into-teaching-app/blob/master/app/helpers/events_helper.rb#L36-L38